src/EnvironmentManagement/EnvironmentManager.py

```diff
@@ -318,6 +328,9 @@ def _assign_players_to_vehicles(self) -> bool:
                         logger.debug('Playtime checker is not needed.')
                     self.update_staff_ui()
                     assigned_any_player = True
+                    for b in self._active_bots:
+                        b.set_is_player_active(True)
+
```

Comment:
is_player_active could be a class attribute. Then you wouldn't need to iterate over all bots